arviz/data/io_cmdstan.py

```diff
@@ -0,0 +1,417 @@
+"""CmdStan-specific conversion code."""
+from collections import defaultdict
+from copy import deepcopy
+from glob import glob
+import linecache
+import os
+import re
+
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from .inference_data import InferenceData
+from .base import requires, dict_to_dataset, generate_dims_coords
+
+
+class CmdStanConverter:
+    """Encapsulate CmdStan specific logic."""
+
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, *, output=None, prior=None, posterior_predictive=None,
+                 observed_data=None, observed_data_var=None,
+                 log_likelihood=None, coords=None, dims=None):
+        self.output = glob(output) if isinstance(output, str) else output
+        if isinstance(output, str) and len(self.output) > 1:
+            msg = "\n".join("{}: {}".format(i, os.path.normpath(path)) \
+                            for i, path in enumerate(self.output, 1))
+            print("glob found {} files for 'output':\n{}".format(len(self.output), msg))
```

Comment:
can we use a logger if we need to print?

Reply:
Do we implement logger?

Reply:
I'm not going to implement logger on this PR. Let's do it in another one.